### PR TITLE
Move classes to their own class files

### PR DESCRIPTION
**Pull request by DanVanAtta**
Not exhaustive, but hits a number of classes that are defined in the same file as  other classes 

This change is [Reviewable](https://reviewable.io/reviews/triplea-game/triplea/931)

### CODE REVIEW

File: src/games/strategy/triplea/delegate/AbstractMoveExtendedDelegateState.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package games.strategy.triplea.delegate;
+
+import java.io.Serializable;
+import java.util.List;
+
+class AbstractMoveExtendedDelegateState implements Serializable {
```
Comment (RoiEXLab) [inline src/games/strategy/triplea/delegate/AbstractMoveExtendedDelegateState.java]: @DanVanAtta Shouldn't this class be public then?

File: src/games/strategy/triplea/delegate/PlaceExtendedDelegateState.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import games.strategy.engine.data.Territory;
+import games.strategy.engine.data.Unit;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+class PlaceExtendedDelegateState implements Serializable {
```
Comment (RoiEXLab) [inline src/games/strategy/triplea/delegate/PlaceExtendedDelegateState.java]: Same here

File: src/games/strategy/triplea/formatter/UnitOwner.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package games.strategy.triplea.formatter;
+
+import games.strategy.engine.data.PlayerID;
+import games.strategy.engine.data.UnitType;
+
+class UnitOwner {
```
Comment (RoiEXLab) [inline src/games/strategy/triplea/formatter/UnitOwner.java]: Here as well

**Comment by RoiEXLab** [comment]
@DanVanAtta Good change overall, but the classes should IMO be public from now on (same for constructor).

**Comment by DanVanAtta** [comment]
Public is not required, I would like to keep this one short and sweet

**Pull request merged**